script/data_cleaning.R

```diff
@@ -65,7 +65,7 @@ dictionary <- data.frame(name = c("Have you seen the exhibition?", "Haben Sie di
 
 # BERLIN DATA ####
 
-(berlin_data <- readr::read_delim(here::here("./data/Ocean._Now_Berlin_-_all_versions_-_labels_-_2021-11-02-15-19-29.csv"), delim = ";",
+(berlin_data <- readr::read_delim(here::here("I:/pwind/Nextcloud/Datascience/OceanNow_data/Ocean._Now_Berlin_-_all_versions_-_labels_-_2021-11-02-15-19-29.csv"), delim = ";",
```

Comment:
This will break to code for me. If we use /data on each of our local systems the code will be reproducible.

```suggestion
(berlin_data <- readr::read_delim(here::here("./data/Ocean._Now_Berlin_-_all_versions_-_labels_-_2021-11-02-15-19-29.csv"), delim = ";",
```